functional-tests/entry.py

```diff
@@ -7,13 +7,46 @@
 from typing import Optional
 
 import flexitest
+from strata_utils import get_address
 
 import factory
 import net_settings
 from constants import *
 from utils import *
 
 
+class BasicLiveEnv(flexitest.LiveEnv):
+    """
+    A common thin layer for all instances of the Environments.
+    """
+
+    def __init__(self, srvs):
+        super().__init__(srvs)
+        self._el_address_gen = (
+            "deada00{:04X}dca3ebeefdeadf001900dca3ebeef".format(x) for x in range(16**4)
+        )
+        self._btc_address_idx = 0
+
+    def gen_el_address(self) -> str:
+        """
+        Generates a unique EL address to be used across tests.
+        """
+        return next(self._el_address_gen)
+
+    def gen_funded_btc_address(self) -> str | List[str]:
+        """
+        Generates a unique bitcoin taproot addresses that is funded with some BTC.
+        """
+        brpc: BitcoindClient = self.svcs["bitcoin"].create_rpc()
+
+        tr_addr: str = get_address(self._btc_address_idx)
+        self._btc_address_idx += 1
+        # 101 to make sure the maturation case is covered when
+        # the Environment is started with no blocks pre-generated.
+        brpc.proxy.generatetoaddress(101, tr_addr)
+        return tr_addr
```

Comment:
This partially solves the issue.
But ultimately we would want a `sendtoaddress` to be exposed in the `sequencer` so that it can send funds to other BTC addresses.

Reply:
We shifted the prios with this a little bit.  We're just going to go with funding them directly and worry about having a sendtoaddress later.